src/us/kbase/narrativejobservice/sdkjobs/SDKMethodRunner.java

```diff
@@ -374,124 +375,121 @@ private static void propagateConfigUrl(Map<String, String> config, String paramN
-            String status = params.getError().getMessage();
-            if (status == null)
-                status = "Unknown error";
-            if (status.length() > 200)
-                status = status.substring(0, 197) + "...";
-            ujsClient.completeJob(ujsJobId, auth.getToken(), status,
-                    params.getError().getError(), null);
-        } else {
-            ujsClient.completeJob(ujsJobId, auth.getToken(), "done", null,
-                    new Results());
-        }
-        // let's make a call to catalog sending execution stats
-		//TODO UNDELETE
-//        try {
-//            final AppInfo info = getAppInfo(ujsJobId, config);
-//            final RunJobParams input = getJobInput(ujsJobId, config);
-//            String[] parts = input.getMethod().split(Pattern.quote("."));
-//            String funcModuleName = parts.length > 1 ? parts[0] : null;
-//            String funcName = parts.length > 1 ? parts[1] : parts[0];
-//            String gitCommitHash = input.getServiceVer();
-//            Long[] execTimes = getAweTaskExecTimes(ujsJobId, config);
-//            long creationTime = execTimes[0];
-//            long execStartTime = execTimes[1];
-//            long finishTime = execTimes[2];
-//            boolean isError = params.getError() != null;
-//            String errorMessage = null;
-//            try {
-//                sendExecStatsToCatalog(auth.getUserName(), info.uiModuleName,
-//                        info.methodSpecId, funcModuleName, funcName,
-//                        gitCommitHash, creationTime, execStartTime, finishTime,
-//                        isError, ujsJobId, config);
-//            } catch (ServerException ex) {
-//                errorMessage = ex.getData();
-//                if (errorMessage == null)
-//                    errorMessage = ex.getMessage();
-//                if (errorMessage == null)
-//                    errorMessage = "Unknown server error";
-//            } catch (Exception ex) {
-//                errorMessage = ex.getMessage();
-//                if (errorMessage == null)
-//                    errorMessage = "Unknown error";
-//            }
-//            if (errorMessage != null) {
-//                String message = "Error sending execution stats to catalog (" +
-//                        auth.getUserName() + ", " + info.uiModuleName + ", " + info.methodSpecId +
-//                        ", " + funcModuleName + ", " + funcName + ", " + gitCommitHash + ", " +
-//                        creationTime + ", " + execStartTime + ", " + finishTime + ", " + isError +
-//                        "): " + errorMessage;
-//                System.err.println(message);
-//                if (log != null)
-//                    log.logErr(message);
-//            }
-//        } catch (Exception ex) {
-//            ex.printStackTrace();
-//            if (log != null)
-//                log.logErr(ex);
-//        }
-    }
+	public static void finishJob(
```

Comment:
Reverting to old version that sends execution stats to the calendar and updates the DB